opensrp-anc/src/main/java/org/smartregister/anc/library/util/Utils.java

```diff
@@ -1163,4 +1163,23 @@ public static String getLocationLocalizedName(Location location, JsonFormActivit
         }
         return locationName;
     }
+
+    public static String extractValuefromJSONObject(String jsonString)
```

Comment:
Add tests for this function for all scenarios. e.g when jsonString is null and when it has value and when the string don't start with { or don't end with }